ansible_base/rbac/validators.py

```diff
@@ -116,6 +116,23 @@ def validate_permissions_for_model(permissions, content_type: Optional[Model], m
                 display_perms = ', '.join(non_add_model_permissions)
                 raise ValidationError({'permissions': f'Permissions for model {role_model._meta.verbose_name} needs to include view, got: {display_perms}'})
 
+    # Check change edit permissions is given for every model that has update/delete/special actions listed
+    if settings.ANSIBLE_BASE_DELETE_REQUIRE_CHANGE:
+        # If role_model is a system-wide role (None means it is), skip this validation.
+        if role_model is not None:
```

Comment:
If you look at the SonarCloud output, you'll probably see it recommend that 2 nested `if` statements should be combined, just as a code style thing